src/Api.ts

```diff
@@ -6,16 +6,21 @@ export class Api {
     private static NE_BASE_ENDPOINT = 'https://gdcportalgw.its-mo.com/api_v181217_NE/gdc';
     private static INITIAL_APP_STRINGS = 'geORNtsZe5I4lRGjG9GZiA';
 
-    private static getBaseEndpoint(regionCode: string) {
-        return (regionCode === 'NNA' ? Api.NNA_BASE_ENDPOINT : Api.NE_BASE_ENDPOINT);
+    private static getBaseEndpoint(regionCode: string, baseEndpoint?: string) {
+        if (typeof baseEndpoint !== 'undefined') {
```

Comment:
Couldn't this just be `if (baseEndpoint !== undefined)` or even `if (baseEndpoint)`?

Reply:
Yes, I'm still used to relying on typeof unnecessarily. New commit added to the PR.